tests/adapter/dbt/tests/adapter/hooks/data/seed_run.sql

```diff
@@ -0,0 +1,16 @@
+
+drop table if exists {schema}.on_run_hook;
+
+create table {schema}.on_run_hook (
```

Comment:
It's minor, but if you always want `on_run_hook` to look like `on_model_hook`, you could just create this table from the other:
```
drop table if exists {schema}.on_run_hook;
create table {schema}.on_run_hook as
select * from {schema}.on_model_hook
where 0 = 1
```